handler/aggregate/loader_test.go

```diff
@@ -76,6 +90,12 @@ var _ = Describe("type Loader", func() {
 				))
 			})
 
+			It("does not return a snapshot", func() {
+				inst, err := loader.Load(ctx, "<handler-key>", "<instance>", base)
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(inst.BarrierEventID).To(BeEmpty()) // is set when snapshot is present
```

Comment:
This line is what clued me into that issue with `md.BarrierEventID` I mentioned above, so I suspect this test will no longer function after that is fixed.